docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -53,44 +55,55 @@ You will need the IP addresses of the nodes in the cluster for the next step.
 {{< /tip>}}
 
 
-### Step 2. TPC-C Load Phase
-
-Before starting the workload, you will need to load the data first. Make sure
-to replace the IP addresses with that of the nodes in the cluster.
+### 2. Configure the benchmark
```

Comment:
Lets change this to `Configure connection params`

Reply:
Done